components/Modal.tsx

```diff
@@ -1,14 +1,20 @@
-import { FC, ReactNode, useId } from 'react'
+import { FC, useId } from 'react'
 import ActionButton, { ActionButtonProps } from './ActionButton'
 import { FocusOn } from 'react-focus-on'
 
 export interface ModalProps {
   actionButtons: ActionButtonProps[]
-  children: ReactNode
   open: boolean
+  header: string
+  description: string
```

Comment:
I would still use `children: ReactNode` instead of description. It gives you more flexibility for the model content, as example if you want to display multiple paragraph.